stations/abc.py

```diff
@@ -12,28 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pytz
-import requests
-from datetime import datetime
+from urllib.request import urlopen
+
+from bs4 import BeautifulSoup
 
 def abc():
     """Custom news fetcher for ABC News Australia briefing"""
-    # Format template with (hour, day, month)
-    url_temp = ('https://abcmedia.akamaized.net/news/audio/news-briefings/'
-                'top-stories/{}{}/NAUs_{}00flash_{}{}_nola.mp3')
-    now = pytz.utc.localize(datetime.utcnow())
-    syd_tz = pytz.timezone('Australia/Sydney')
-    syd_dt = now.astimezone(syd_tz)
-    hour = syd_dt.strftime('%H')
-    day = syd_dt.strftime('%d')
-    month = syd_dt.strftime('%m')
-    year = syd_dt.strftime('%Y')
-    url = url_temp.format(year, month, hour, day, month)
-
-    # If this hours news is unavailable try the hour before
-    response = requests.get(url)
-    if response.status_code != 200:
-        hour = str(int(hour) - 1)
-        url = url_temp.format(year, month, hour, day, month)
-
-    return url
+    domain = "https://www.abc.net.au"
+    latest_briefings_url = f"{domain}/radio/newsradio/news-briefings/"
+    soup = BeautifulSoup(urlopen(latest_briefings_url), features='html.parser')
+    result = soup.find(id="collection-grid3")
+    episode_page_link = result.find_all('a')[0]['href']
```

Comment:
what do `a`, `[0]`, and `href` represent?  can we make this a little less cryptic?

Reply:
Added comments, which is kind of the best we can do here I think. Do the two comments above make it clear enough?